js/sound-player.js

```diff
@@ -1,13 +1,19 @@
-const playButton = document.querySelector('.player__button--play');
-const pauseButton = document.querySelector('.player__button--pause');
+const volumeOn = document.querySelector('.fa-volume-up');
```

Comment:
this is a small app so the variable naming probably wont get confusing, but it's always a good idea to be descriptive with naming in your app code. 
`const volumeOnIconElement` could work here too for instance.